testdata/baselines/reference/submoduleAccepted/conformance/importTag16.types.diff

```diff
@@ -0,0 +1,12 @@
+--- old.importTag16.types
++++ new.importTag16.types
+@@= skipped -12, +12 lines =@@
+  * @param {I} b
+  */
+ export function foo(a, b) {}
+->foo : (a: Foo, b: I) => void
+->a : Foo
++>foo : (a: default, b: I) => void
```

Comment:
I don't know how typescript behaves, but I've changed so little code that I guess that `a: default` is closer to TS than `a: Foo` was.

Reply:
Isn’t this saying that TS has `a: Foo` where tsgo has `a: default`? We definitely shouldn’t be calling that type `default`.

Reply:
Oh, yeah, I misread this as diffing from the previous version of Strada, but the fact that it's a new diff file means that the type output was identical before -- even though the error output was different.

Reply:
This is a limitation of our type printer. Here's the equivalent TS output, which is wrong in the same way:

```
=== b.ts ===
import type Foo, { type I } from "./a"
>Foo : default
>I : I

/**
 */
export function foo(a: Foo, b: I) {}
>foo : (a: default, b: I) => void
>a : default
>b : I
```